bgpd/bgp_routemap.c

```diff
@@ -4401,8 +4403,12 @@ DEFUN (set_lcommunity_delete,
        "Large Community-list name\n"
        "Delete matching large communities\n")
 {
+	int idx_lcomm_list = 2;
+	char *args;
+
+	args = argv_concat(argv, argc, idx_lcomm_list);
```

Comment:
This function allocates memory that needs to be freed with MTYPE_TMP after use.

Reply:
fixed